fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -457,8 +457,14 @@ private long getTimestamp(final String property, final long created) throws Repo
         LOGGER.trace("Using {} date", property);
         final long timestamp = getProperty(property).getDate().getTimeInMillis();
         if (timestamp < created && created > NO_TIME && !isFrozenResource()) {
-            LOGGER.trace("Updating {} with later created date", property);
-            getNode().setProperty(property, created);
+            try {
+                LOGGER.trace("Updating {} with later created date", property);
+                getNode().setProperty(property, created);
+            } catch (final javax.jcr.AccessDeniedException e) {
+                LOGGER.trace("Unable to update {} with the later created date", property);
```

Comment:
So that's the meat of the issue here... this method (getTimestamp) is supposed to GET a value.  It seems we also use it to correct data in the repository.  That would be fine if this method were only invoked by operations that are expected (and authorized) to update the resource in question (though, I'd hope the name changed to reflect that it actually updates the repository).  

The following execution path will likely illustrate the problem:

	at org.fcrepo.kernel.modeshape.FedoraResourceImpl.getTimestamp(FedoraResourceImpl.java:462)
	at org.fcrepo.kernel.modeshape.FedoraResourceImpl.getLastModifiedDate(FedoraResourceImpl.java:437)
	at org.fcrepo.kernel.modeshape.FedoraResourceImpl.getEtagValue(FedoraResourceImpl.java:758)
	at org.fcrepo.http.api.ContentExposingResource.addCacheControlHeaders(ContentExposingResource.java:502)
	at org.fcrepo.http.api.ContentExposingResource.checkCacheControlHeaders(ContentExposingResource.java:481)
	at org.fcrepo.http.api.ContentExposingResource.getBinaryContent(ContentExposingResource.java:317)
	at org.fcrepo.http.api.ContentExposingResource.getContent(ContentExposingResource.java:202)
	at org.fcrepo.http.api.FedoraLdp.getResource(FedoraLdp.java:228)

In cases where a user makes an HTTP GET, we end up invoking this code, and in some cases, with access controls enabled, this update is disallowed.  

Before the change proposed by this PR, that AccessDeniedException was thrown and later caught somewhere in the call stack and manifested as a 403.  That was extremely difficult to debug, since there are so many places where authorization checks occur.